source/collections/VectorList.ooc

```diff
@@ -135,12 +140,19 @@ VectorList: class <T> {
 			result add(this[indices[i]])
 		result
 	}
-	getSlice: func (range: Range) -> This<T> {
-		result := This<T> new(range count)
-		this getSliceInto(range, result&)
-		result
+	getSlice: func (start, end: Int) -> This<T> {
+		result := This<T> new(end - start + 1)
+		this getSliceInto(start, end, result&)
+	}
```

Comment:
Put `getSlice: func (range: Range) -> This<T> {` back and have `getSlice: func (start, end: Int) -> This<T> {` do this instead:
```ooc
getSlice: func (start, end: Int) -> This<T> {
    this getSliceInto(start .. end)
}
```